superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/components/RadioButtonControl.tsx

```diff
@@ -53,8 +53,12 @@ export default function RadioButtonControl({
         '.btn:focus': {
           outline: 'none',
         },
+        '.control-label': {
+          color: theme.colors.grayscale.base,
+          marginBottom: theme.gridUnit,
+        },
         '.control-label + .btn-group': {
-          marginTop: 1,
+          marginTop: theme.gridUnit * 0.25,
```

Comment:
I think it's fine to avoid fractional gridUnits, and just use 1px in this case.... if it's even needed at all. 
```suggestion
          marginTop:  1px,
```